src/index.tsx

```diff
@@ -879,6 +887,9 @@ class DraggableFlatList<T> extends React.Component<Props<T>, State> {
         >
           <AnimatedFlatList
             {...this.props}
+            ListHeaderComponent={listHeader}
```

Comment:
why is this necessary? props get spread onto the list, so you should just be able to pass `ListHeaderComponent` as a prop to `DraggableFlatlist`

Reply:
oh, i didn't notice that, lol, my bad